Resources/Server Info/RP_Rules.txt

```diff
@@ -1,103 +1,119 @@
+
+[color=#a4885c]11.[/color] Don't be a dick. You are playing a multiplayer game with other people who also want to enjoy the game.
+    - Do not intentionally make other players' lives hell for your own amusement.
+    - Antagonists have lots of leeway with this rule and may kill/sabotage as they see fit, but if it degrades the experience for majority of the rest of the server you will be told to stop.
+    - [color=#ff0000]THE ROUND IS NOT OVER UNTIL THE END-ROUND SUMMARY APPEARS. KILLING SOMEONE FOR NO REASON BEFORE THIS WILL BE HANDLED ACCORDINGLY.[/color]
+
+[color=#a4885c]12.[/color] Don't harass or target players across rounds for actions in prior rounds or for actions outside of the game (this is referred to as "Metagrudging".) 
+    - Annoying players for IC reasons in the current round is fine; doing it across rounds or as a ghost role after they kill you is not.
+
+[color=#a4885c]13.[/color] Don't use information gained from outside your character's knowledge to gain an advantage (this is referred to as "Metagaming").
+    - Using information you gain from outside your own character (such as spectating while a ghost, through screenshots or metacomms, or other means) to your advantage is strictly forbidden. You remember all events up until you fall unconscious, even after cloning. If you take a ghost role, unless otherwise stated, you DO NOT REMEMBER ANYTHING from your past life.
+
+[color=#a4885c]14.[/color] Follow escalation rules and don't murder someone for slipping you, use common sense.
+    - You can defend yourself to the extent of protecting your own life.
+    - Security is unique in escalation and can escalate to less lethal to arrest criminals as needed.
+    - Don't outright leave people to die if you get in a fight, make an effort to heal them or bring them to Medbay. A
+    - Adminhelp the situation if you think someone is over-escalating.
+    - Department strikes, revolutions (ex: cargonia and any variation thereof), or cults are antagonist-only activites and are forbidden.
+
+[color=#a4885c]15.[/color] Don't immediately ghost or suicide from your role if you do not get antagonist (referred to as "Antag-rolling").
+    - This is not fair to other players actually waiting patiently for an antagonist round. Alternatively, if you do not want to play an antagonist or do not want to cause conflict (ex: friendly antagonists), do not opt-in for antagonist roles.
+
+[color=#a4885c]16.[/color] Don't rush for or prepare equipment unrelated to your job for no purpose other then to have it "just in case" (referred to as "Powergaming").
+    - A medical doctor does not need insulated gloves, and the Head of Personnel does not need to give themselves armory access so they can go grab a gun.
+    - Don't manufacture weapons, bombs, or death poisons before you know of any reason you would need them.
```

Comment:
I'm seeing some ambiguity here, would this mean that Science isn't allowed to make these items? In the overall rule it makes it seem like it's OK to create items that are related to your job but then goes on to say NOT to prepare Bombs/Weapons/Poisons. This would be taking away a major aspect of Science and the overall Sandbox if that was the case.